### PR TITLE
Scenarios 5.1.1, 5.2.1 and 7.1.1

### PR DESCRIPTION
**Pull request by VitaliiTytarenko-okta**

### CODE REVIEW

File: samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/Login.java [hunk 1]
```diff
@@ -83,6 +83,16 @@ public void refresh_token_stored() {
 		Assert.assertFalse("Can't access refresh_token", refreshToken.isEmpty());
 	}
 
+	@And("^the cell for the value of \"email\" is shown and contains her email$")
+	public void the_cell_for_email_is_shown_and_contains_her_email() {
+		Assert.assertTrue(rootPage.email.isDisplayed());
```
Comment (JayNewstrom) [inline samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/Login.java:88]: I think we should add some more validations here: We're missing the part `and contains her email`

File: samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java [hunk 2]
```diff
@@ -138,6 +146,14 @@ public void assignMFAGroupBeforeScenario(Scenario scenario) {
 		groupList.forEach(group -> Page.getUser().addToGroup(group.getId()));
 	}
 
+	@After("@requireMFAGroupsForUser")
```
Comment (JayNewstrom) [inline samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java:149]: If I understand what this is doing, it's removing the user from the everyone group. Does that work?

In standups, I think we discussed having 2 different facebook users, so we don't have to do this.

Comment (VitaliiTytarenko-okta) [reply samples/embedded-auth-with-sdk/src/test/java/env/Hooks.java:149]: Yes, this is the first solution that made before the discussion. Vijet already created a separate facebook user with MFA and I'll fix this.

File: samples/embedded-auth-with-sdk/src/test/java/pages/Page.java [hunk 3]
```diff
@@ -30,7 +30,7 @@
 
 public class Page {
 
-    private static final int RETRY_COUNT = 5;
+    private static final int RETRY_COUNT = 10;
```
Comment (JayNewstrom) [inline samples/embedded-auth-with-sdk/src/test/java/pages/Page.java:33]: I think most other SDKs are retrying between 30-90 times. But I think by adding resend code functionality to our test logic, we would make these tests more reliable. See conversation here: https://okta.slack.com/archives/G01G626KYBA/p1623354657050900

File: samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/Login.java [hunk 4]
```diff
@@ -83,6 +83,17 @@ public void refresh_token_stored() {
...
 
+	@And("the cell for the value of \"email\" is shown and contains her {word}")
+	public void the_cell_for_email_is_shown_and_contains_her_email(String email) {
+		Assert.assertTrue(rootPage.email.isDisplayed());
+		Assert.assertEquals(System.getenv(email), rootPage.email.getText());
+	}
+
+	@And("^the cell for the value of \"name\" is shown and contains her first name and last name$")
+	public void the_cell_for_name_is_shown_and_contains_her_name() {
+		Assert.assertTrue(rootPage.name.isDisplayed());
```
Comment (JayNewstrom) [inline samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/Login.java:94]: We should also validate the other half of this statement. `and contains her first name and last name`

**Review by JayNewstrom — APPROVED** [review]
Looks good. But I do think we should still add the last validation.

**Review by JayNewstrom — APPROVED** [review]

**Pull request merged**